docs/using/selecting.md

```diff
@@ -43,7 +43,7 @@ repository. The following sections describe these images including their content
 
 * Everything in `jupyter/base-notebook`
 * [Pandoc](http://pandoc.org) and [TeX Live](https://www.tug.org/texlive/) for notebook document conversion
-	* [git](https://git-scm.com/), [emacs](https://www.gnu.org/software/emacs/), [vi](actually [vim-tiny](https://vim.org)), [jed](https://www.jedsoft.org/jed/), [nano](https://www.nano-editor.org/), tzdata, and
+* [git](https://git-scm.com/), [emacs](https://www.gnu.org/software/emacs/), [vi](actually [vim-tiny](https://vim.org)), [jed](https://www.jedsoft.org/jed/), [nano](https://www.nano-editor.org/), tzdata, and
```

Comment:
Thanks for the tab, however as I said in my previous comment the vim entry does not render correctly, see the screenshot below. Could you please fix it? I've made a proposal in my previous comment.
<img width="454" alt="add_vim_to_minimal-notebook__cf__issue_779__by_peterprescott_·_Pull_Request__1059_·_jupyter_docker-stacks" src="https://user-images.githubusercontent.com/7131913/78633013-c301d880-78a0-11ea-8bcd-3d2e58f0a118.png">
Many thanks.